pkg/assembler/assembler.go

```diff
@@ -43,6 +43,7 @@ type IngestPredicates struct {
 	HashEqual        []HashEqualIngest        `json:"hashEqual,omitempty"`
 	PkgEqual         []PkgEqualIngest         `json:"pkgEqual,omitempty"`
 	Vex              []VexIngest              `json:"vex,omitempty"`
+	PointOfContact   []PointOfContactIngest   `json:"contact,omitempty"`
```

Comment:
can you make sure that the use of this is also added? i would take a look at where IngestPredicates is used in the code base and add a case for PointOfContact